main.go

```diff
@@ -8,8 +8,13 @@ import (
 )
 
 func main() {
-	eventServer := server.NewEventServer(storage.NewEventStorage())
-	metricServer := server.NewMetricsServer(storage.NewEventStorage())
+
+	store, err := storage.NewDbStorage("root:123456@tcp(127.0.0.1:3307)/calendar?charset=utf8mb4", 60, 10)
```

Comment:
This is a goog place to use config. For DB address and pass. Can be done later on